homeassistant/components/traccar/const.py

```diff
@@ -0,0 +1,31 @@
+"""Constants for Traccar platform."""
```

Comment:
Drop "platform" here or change it to "integration"

Reply:
Fixed